src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js

```diff
@@ -203,4 +203,25 @@ export default class AircraftTypeDefinitionModel extends BaseModel {
 
         return aircraftIcao.toUpperCase();
     }
+    /**
+     * Checks if the given speed is valid.
+     *
+     * @for AircraftTypeDefinitionModel
+     * @param nextSpeed
+     * @return {Boolean}
+     */
+     isValidSpeed(nextSpeed) {
```

Comment:
This method name doesn't seem appropriate to me. What really makes a "speed" to be "valid" or not? I would say it's whether or not it is a number. Whether or not `nextSpeed` is _valid_ isn't really the question this method answers... perhaps a better name would be like `isAttainableSpeed()`?